<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 126, in main
    result = linter.lint(recipes, subdir, args.variant_config_files, args.exclusive_config_files, args.fix)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 634, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 734, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 244, in from_file
    raise exc
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 235, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 167, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 284, in render
    self.meta = renderer.render(self.recipe_dir, self.dump(), self.selector_dict, self.renderer)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 235, in render
    return ruamel.load(yaml_text.replace("\t", " ").replace("%", " "))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/main.py", line 434, in load
    return constructor.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 119, in get_single_data
    node = self.composer.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 76, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 99, in compose_document
    node = self.compose_node(None, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 223, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 223, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 141, in compose_node
    node = self.compose_sequence_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/composer.py", line 184, in compose_sequence_node
    while not self.parser.check_event(SequenceEndEvent):
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/parser.py", line 146, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/parser.py", line 530, in parse_block_sequence_entry
    if self.scanner.check_token(BlockEntryToken):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/scanner.py", line 1794, in check_token
    while self.need_more_tokens():
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/scanner.py", line 211, in need_more_tokens
    self.stale_possible_simple_keys()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/ruamel/yaml/scanner.py", line 360, in stale_possible_simple_keys
    raise ScannerError(
ruamel.yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 42, column 1:
    All that fail in test_common.py
    ^ (line: 42)
could not find expected ':'
  in "<unicode string>", line 55, column 1:
    # All that fail in py<311 in oth ...
    ^ (line: 55)

ERROR conda.cli.main_run:execute(49): `conda run conda-lint /tmp/abs_bdbhck4ki_/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -29,7 +29,7 @@ requirements:
     - setuptools
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=1.13.3
     - scipy >=0.19.1
     - scikit-learn >=0.20.0
     - matplotlib-base >=2.0.0

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -38,39 +38,38 @@ requirements:
 # See the many issues that also affects tests, and comments
 # that mention that the package is not maintained e.g.:
 #   https://github.com/scikit-optimize/scikit-optimize/issues/1137
-#   https://github.com/scikit-optimize/scikit-optimize/issues/1137
 #
 # All that fail in test_common.py
-# {% set tests_to_skip = "test_per_second_api" %}
-# {% set tests_to_skip = tests_to_skip + " or test_early_stopping_delta_x" %}
-# {% set tests_to_skip = tests_to_skip + " or test_consistent_x_iter_dimensions" %}
-# {% set tests_to_skip = tests_to_skip + " or test_repeated_x" %}
-# {% set tests_to_skip = tests_to_skip + " or test_mixed_spaces" %}
-# {% set tests_to_skip = tests_to_skip + " or test_categorical_init_vals" %}
-# {% set tests_to_skip = tests_to_skip + " or test_init_vals" %}
-# {% set tests_to_skip = tests_to_skip + " or test_init_points_and_models" %}
-# {% set tests_to_skip = tests_to_skip + " or test_init_vals_and_models" %}
-# {% set tests_to_skip = tests_to_skip + " or test_minimizer_with_space" %}
-# {% set tests_to_skip = tests_to_skip + " or test_fixed_random_states" %}
-# {% set tests_to_skip = tests_to_skip + " or test_minimizer_api" %}
-# # All that fail in py<311 in other test modules
-# {% set tests_to_skip = tests_to_skip + " or test_reproducible_runs" %}        # [py<311]  
-# {% set tests_to_skip = tests_to_skip + " or test_same_set_of_points_ask" %}   # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_all_points_different" %}     # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_constant_liar_runs" %}       # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_optimizer_base_estimator_string_smoke" %}  # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_exhaust_initial_calls" %}    # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_optimizer_copy" %}           # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_acq_optimizer_with_time_api" %}  # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_returns_result_object" %}    # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_model_queue_size" %}         # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_multiple_asks" %}            # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_tree_based_minimize" %}      # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_categorical_integer" %}      # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_dump_and_load_optimizer" %}  # [py<311]
-# {% set tests_to_skip = tests_to_skip + " or test_deadline_stopper" %}  # [win]
-# {% set tests_to_skip = tests_to_skip + " or test_timer_callback" %}    # [win]
-# {% set tests_to_skip = tests_to_skip + " or test_real" %}              # [linux-64]
+{% set tests_to_skip = "test_per_second_api" %}
+{% set tests_to_skip = tests_to_skip + " or test_early_stopping_delta_x" %}
+{% set tests_to_skip = tests_to_skip + " or test_consistent_x_iter_dimensions" %}
+{% set tests_to_skip = tests_to_skip + " or test_repeated_x" %}
+{% set tests_to_skip = tests_to_skip + " or test_mixed_spaces" %}
+{% set tests_to_skip = tests_to_skip + " or test_categorical_init_vals" %}
+{% set tests_to_skip = tests_to_skip + " or test_init_vals" %}
+{% set tests_to_skip = tests_to_skip + " or test_init_points_and_models" %}
+{% set tests_to_skip = tests_to_skip + " or test_init_vals_and_models" %}
+{% set tests_to_skip = tests_to_skip + " or test_minimizer_with_space" %}
+{% set tests_to_skip = tests_to_skip + " or test_fixed_random_states" %}
+{% set tests_to_skip = tests_to_skip + " or test_minimizer_api" %}
+# All that fail in py<311 in other test modules
+{% set tests_to_skip = tests_to_skip + " or test_reproducible_runs" %}        # [py<311]  
+{% set tests_to_skip = tests_to_skip + " or test_same_set_of_points_ask" %}   # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_all_points_different" %}     # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_constant_liar_runs" %}       # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_optimizer_base_estimator_string_smoke" %}  # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_exhaust_initial_calls" %}    # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_optimizer_copy" %}           # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_acq_optimizer_with_time_api" %}  # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_returns_result_object" %}    # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_model_queue_size" %}         # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_multiple_asks" %}            # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_tree_based_minimize" %}      # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_categorical_integer" %}      # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_dump_and_load_optimizer" %}  # [py<311]
+{% set tests_to_skip = tests_to_skip + " or test_deadline_stopper" %}  # [win]
+{% set tests_to_skip = tests_to_skip + " or test_timer_callback" %}    # [win]
+{% set tests_to_skip = tests_to_skip + " or test_real" %}              # [linux-64]
 
 
 test:
@@ -90,7 +89,7 @@ test:
     - export SKOPT_SKIP_NETWORK_TESTS=1  # [not win]
     - set SKOPT_SKIP_NETWORK_TESTS=1     # [win]
     # - pytest --showlocals --pyargs -m "not fast_test" -m "not slow_test" -m "not slow" --ignore=skopt/tests/test_benchmarks.py -k "not({{ tests_to_skip }})" skopt/tests
-    - pytest --showlocals --pyargs -m "not fast_test" -m "not slow_test" -m "not slow" skopt/tests
+    - pytest --showlocals --pyargs -m "not fast_test" -m "not slow_test" -m "not slow" skopt/tests -k "not({{ tests_to_skip }})" skopt/tests
 
 about:
   home: https://scikit-optimize.github.io/

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -9,7 +9,9 @@ source:
   url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
   sha256: b5602b22c7aacabe3afd36619f662d1f287f2e3fe9a5aa19bbc83f851b30b7bf
   patches:
+    # this patch fixes the use of deprecated np.int (numpy >=1.20)
     - 0001-replace-deprecated-np.int.patch
+    # this patch fixes an incorrectly chosen error rounding criteria used in the required scikit-learn version
     - 0002-use-compatible-mse-criteria-with-scikit-learn.patch
 
 build:
@@ -86,9 +88,10 @@ test:
     - skopt/tests
   commands:
     - pip check
+    # this var is set in the upstream project
     - export SKOPT_SKIP_NETWORK_TESTS=1  # [not win]
     - set SKOPT_SKIP_NETWORK_TESTS=1     # [win]
-    # - pytest --showlocals --pyargs -m "not fast_test" -m "not slow_test" -m "not slow" --ignore=skopt/tests/test_benchmarks.py -k "not({{ tests_to_skip }})" skopt/tests
+    # pytest parameters '-m "not fast_test" -m "not slow_test" -m "not slow"'' are set in the upstream project
     - pytest --showlocals --pyargs -m "not fast_test" -m "not slow_test" -m "not slow" skopt/tests -k "not({{ tests_to_skip }})" skopt/tests
 
 about:

